src/content.js

```diff
@@ -1,220 +1,161 @@
-const glob = require("glob");
-const renderPage = require("./render/render");
-const {findHeaders} = require("./render/components");
 const marked = require("marked");
-const buildData = require("./data");
-const fm = require("front-matter");
+const yaml = require("js-yaml");
 const fs = require("fs").promises;
 const path = require("path");
-const MiniSearch = require("minisearch");
+const R = require("ramda");
 
-const STOP_WORDS = new Set(["and", "or", "not", "to", "at", "in", "a", "the", "be", "are", "is", "as", "its", "it", "this", "these", "any", "halo", "e", "g"]);
+const {commonLength, findPaths} = require("./utils");
+const loadStructuredData = require("./data");
+const renderPages = require("./build/render");
+const buildSitemap = require("./build/sitemap");
+const buildResources = require("./build/resources");
+const buildSearchIndex = require("./build/search");
```

Comment:
The responsibilities of `content.js`, which builds all the content/pages, have been split up into various build tasks. This file is now mainly responsible for indexing all the pages, loading data, and orchestrating the build.